app/workers/postman_worker.rb

```diff
@@ -0,0 +1,8 @@
+class PostmanWorker
+  include Sidekiq::Worker
+
+  def perform(h, count)
+    h = JSON.load(h)
+    VisitorMailer.contact_email(h['name'], h['email'], h['message']).deliver
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.